src/prefabs/structures/CheckboxGroup/index.ts

```diff
@@ -37,6 +39,10 @@ export const CheckboxGroup = (
     },
   ];
 
+  if (config.value) {
+    options.value = updateOption(options.value, { ...config.value });
```

Comment:
We do we set the config value into options?

Reply:
so that you can reuse the component for other prefabs and overwrite the default value and its settings like the allowedKinds